tests/test_sqlalchemy_data_layer.py

```diff
@@ -767,14 +803,41 @@ def test_get_list_with_simple_filter(client, register_routes, person, person_2):
         querystring = urlencode(
             {
                 "page[number]": 1,
-                "page[size]": 1,
+                "page[size]": 10,
                 "fields[person]": "name,birth_date",
                 "sort": "-name",
                 "filter[name]": "test",
             }
         )
         response = client.get("/persons" + "?" + querystring, content_type="application/vnd.api+json")
         assert response.status_code == 200
+        assert len(response.json['data']) == 1
```

Comment:
will it be `1` always if page size is changed (above)?

Reply:
changed to meta count